src/main/scala/net/snowflake/spark/snowflake/streaming/SnowflakeSink.scala

```diff
@@ -125,6 +125,7 @@ class SnowflakeSink(sqlContext: SQLContext,
         val time = System.currentTimeMillis()
         metric.put(END_TIME, time)
         metric.get(LOAD_RATE).asInstanceOf[ObjectNode].put(END_TIME, time)
+        SnowflakeTelemetry.addCommonFields(metric)
```

Comment:
do we still maintain this stream connector?

Reply:
We don't officially support new features. It's not necessary to maintain it. Since it is some lines low risk change, it is better to maintain it.